openshift-storage-libs/openshiftstoragelibs/baseclass.py

```diff
@@ -1459,10 +1459,13 @@ def validate_glusterfsd_memory_usage(self, size_limit):
         for gluster_node in self.gluster_servers:
             # Fetch pid from the node/pod
             out = cmd_run_on_gluster_pod_or_node(
-                self.ocp_master_node[0], get_glusterfsd_pid, gluster_node)
-            self.assertTrue(
-                out, "Failed to get pid of glusterfsd from node/pod "
-                "{}".format(gluster_node))
+                self.ocp_master_node[0],
+                get_glusterfsd_pid, gluster_node,
+                raise_on_error=False)
+            if not out:
+                g.log.info("'glusterfsd'process is not present on node/pod "
+                    "{}".format(gluster_node))
+                continue
```

Comment:
`continue` not required

Reply:
Yes, it's required. Continue will skip next  steps from the loop and move to next item. If we remove it, `out` will have value `None` which will raise  exception at line  #1469